File: week1/이상현/7.11/A006.cpp
#include <string>
#include <iostream>
using namespace std;

bool solution(string s)
{
    bool answer = true;
    int count_p = 0;
    int count_y = 0;
    
    // [실행] 버튼을 누르면 출력 값을 볼 수 있습니다.
    for(int i=0; i<s.length(); i++){
        if(s[i] == 'p' || s[i] == 'P'){
            count_p++;
           
        }
        else if(s[i] == 'y' || s[i] == 'Y'){
            count_y++;
           
        }
    }
    if(count_p == count_y){
        answer = true;
    }
    else{
        answer = false;
    }

    return answer;
}

File: week1/김희라/A006.java
class Solution {
  boolean solution(String s) {
    boolean answer = true;

    s = s.toLowerCase();

    int cntP = 0;
    int cntY = 0;

    for (int i = 0; i < s.length(); i++) {
      char a = s.charAt(i);
      if (a == 'p')
        cntP++;
      else if (a == 'y')
        cntY++;
    }

    if (cntP != cntY)
      answer = false;

    return answer;
  }
}